tests/functional/test_smoke_tests.py

```diff
@@ -0,0 +1,75 @@
+from .utils import helpers
+import pytest
+import uuid
+from tests.functional.test_user_restricted import (
+    AUTH_HEALTHCARE_WORKER,
+    retrieve_scenario,
+    search_scenario,
+    related_person_scenario,
+    status_scenario
+)
+
+from pytest_nhsd_apim.auth_journey import AuthorizationCodeConfig, AuthorizationCodeAuthenticator
+
+pytestmark = pytest.mark.smoke_test
+
+
+@retrieve_scenario('Healthcare worker can retrieve patient')
+def test_retrieve_patient():
+    pass
+
+
+@search_scenario('Healthcare worker can search for patient')
+def test_search_patient():
```

Comment:
are these smoke tests for future development i.e. markers?

Reply:
The tests in test_smoke_tests are the same tests that were previously in test_user_restricted.py with the marker `smoke_tests`. Due to the process of producing auth headers in the pytest-nhsd-apim library, they need to sit in a different file.